From a1aacd718650dc7f1d2bbb4a652d20f00a1e7444 Mon Sep 17 00:00:00 2001
From: Meng Dongyang <daniel.meng@rock-chips.com>
Date: Mon, 8 Jan 2018 18:09:27 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3326-evb-lp3-v10: enable USB
 controllers

Enable DWC2 controller, EHCI controller, OHCI controller and
USB PHY.

Change-Id: I6c1fe5f9f074352f2a18db6337a855cf7d8089f2
Signed-off-by: Meng Dongyang <daniel.meng@rock-chips.com>
---
 .../boot/dts/rockchip/rk3326-evb-lp3-v10.dts  | 24 +++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
index 2029d3dc1e62..b0f177aa6a3c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3326-evb-lp3-v10.dts
@@ -378,6 +378,30 @@
 	status = "disabled";
 };
 
+&u2phy {
+	status = "okay";
+
+	u2phy_host: host-port {
+		status = "okay";
+	};
+
+	u2phy_otg: otg-port {
+		status = "okay";
+	};
+};
+
+&usb20_otg {
+	status = "okay";
+};
+
+&usb_host0_ehci {
+	status = "okay";
+};
+
+&usb_host0_ohci {
+	status = "okay";
+};
+
 &vopb {
 	status = "okay";
 };
-- 
2.35.3

